db/db_postgres.c

```diff
@@ -93,7 +92,7 @@ static PGresult *db_postgres_do_exec(struct db_stmt *stmt)
 
 		switch (b->type) {
 		case DB_BINDING_UNINITIALIZED:
-			db_fatal("DB binding not initialized: position=%zu, "
+			db_fatal(1, "DB binding not initialized: position=%zu, "
```

Comment:
```suggestion
			db_fatal(GENERIC_DB_ERROR, "DB binding not initialized: position=%zu, "
```

Do you think that it is the right moment to make a small refactoring in the error code? I started to put the error code in the `common/error.h` file, to make an easy life to catch the reason for failure from a bash script, and I think maybe we can put also some DB error?

Reply:
No I don't think that this is the right time to do refactoring here but a follow up PR would be most welcome.

I'm not sure that pulling the database error code out is the right idea -- it's specifically separate to allow for catching errors in the *database* and therefore isn't a good candidate for addition to `common` -- keeping it in `db` is the correct choice here imo